### ISSUE TITLE
2-of-2 Multisig Timelock decaying to single

### ISSUE THREAD

**Issue by xsats** [photon-sdk/photon-lib#28]
Upon my first discovery of the photon-sdk, the keyserver architecture reminded me a bit of the way [Blockstream Green's standard account](https://help.blockstream.com/hc/en-us/articles/900001391763-How-does-Blockstream-Green-s-multisig-security-work-#h_a0e00982-eb22-42f9-af2f-997bf0f67a09) functions.
One major difference between the two is Blockstream's CSV timelock, where utxos received in the wallet decay to a single sig after a year, permitting recovery of funds in the event of loss of user access to their wallet/device. I understand that this would mean that the keyserver would need to play a much more active role during wallet usage, since it would then be required to sign any transactions inside the timelock period, but I am curious to know whether this is something that you have considered for Photon and hear more thoughts from others on whether this might be a beneficial addition for this project.

**Comment by moneyball** [comment]
I agree this is a good feature and could fit the scope of Photon. The only question in my mind is the priority of this vs. other initiatives, such as:
1. Providing native APIs for Kotlin/Android and Swift/iPhone
2. Security audit/more eyeballs on the code to gain confidence in using it in production at scale
3. Adding support for multi-sig wallet backup data (pubkeys)
4. Adding support for LN wallet backup data
